mslib/msui/constants.py

```diff
@@ -34,17 +34,33 @@
 # ToDo refactor to generic functions, keep only constants
 HOME = os.path.expanduser(f"~{os.path.sep}")
 MSUI_CONFIG_PATH = os.getenv("MSUI_CONFIG_PATH", os.path.join(HOME, ".config", "msui"))
+_fs = None
 if '://' in MSUI_CONFIG_PATH:
     try:
-        _fs = fs.open_fs(MSUI_CONFIG_PATH)
-    except fs.errors.CreateFailed:
-        _fs.makedirs(MSUI_CONFIG_PATH)
+        _fs = fs.open_fs(fs.path.dirname(MSUI_CONFIG_PATH))
```

Comment:
I am not convinced that fs.path.dirname is correct there. For the default value of `~/.config/msui` defined above this would open `~/.config`, which seems wrong to me.